src/content/rohlik.tsx

```diff
@@ -1,5 +1,12 @@
 import { Rohlik } from "../components/Rohlik";
 import { render } from "./render";
-import {HashRouter as Router} from "react-router-dom";
+import { HashRouter as Router } from "react-router-dom";
 
-render(<Router><Rohlik /></Router>);
\ No newline at end of file
+// TODO: It looks that if we bootstrap the app too fast, the original client gets confused because hydration doesn't match.
+setTimeout(() => {
```

Comment:
Tohle bych tady ted nechal, startuje mi to pak lip, jinak tam mam zvlastni chyby z "jejich" reactu. Nevim jestli se ti to stava taky, ale tohle je hotfix.